src/explore-education-statistics-admin/src/modules/find-statistics/PublicationReleaseContent.tsx

```diff
@@ -33,6 +33,10 @@ const PublicationReleaseContent = () => {
     EditableBlock[]
   >(release.summarySection.content);
 
+  useEffect(() => {
+    setSummarySectionBlocks(release.summarySection.content);
```

Comment:
This is the wrong way to go about this. Pass `release.summarySection.content` directly to the `EditableSectionBlocks`.

Reply:
I thought as much, but I think you removed this as part as some performance improvement - so I didn't want to undo that. I'll fix up as you've suggested, thanks